include/swift/AST/Expr.h

```diff
@@ -180,11 +180,12 @@ class alignas(8) Expr : public ASTAllocated<Expr> {
     LiteralCapacity : 32
   );
 
-  SWIFT_INLINE_BITFIELD(DeclRefExpr, Expr, 2+2+1+1,
+  SWIFT_INLINE_BITFIELD(DeclRefExpr, Expr, 2+2+1+1+1,
     Semantics : 2, // an AccessSemantics
     FunctionRefKind : 2,
     IsImplicitlyAsync : 1,
-    IsImplicitlyThrows : 1
+    IsImplicitlyThrows : 1,
+    IsDistributedThunkCall: 1
```

Comment:
not used yet; but I guess we could...? do we have to set it on the `DeclRefExpr`, or do we somehow get the `ApplyExpr` in the `  void visitDeclRefExpr(DeclRefExpr *e) {`?

Either way, we're currently setting on the ApplyExpr, but in SILGen we deal with the DeclRefExpr so we need to fix something 🤔 

Technically "implicitly throws" is enough but might be nicer to have a separate flag?